core/eolearn/core/__init__.py

```diff
@@ -28,8 +28,8 @@
     MergeEOPatchesTask,
 )
 
-from .fs_utils import get_filesystem, load_s3_filesystem
-from .utilities import deep_eq, negate_mask, constant_pad, get_common_timestamps, bgr_to_rgb, FeatureParser
-
+from .utilities.filesystem import get_filesystem, load_s3_filesystem
```

Comment:
I suggest renaming:

- folder `utilities` -> `utils` (we have same in `eo-grow`),
- module `filesystem` -> `fs` (we have same in `eo-grow` and its shorter),
- module `feature_parser` -> `parsing`
- module `other` -> `common`

Reply:
I used `filesystem` because `fs` was flagged by my linters as a self import (since we use the `fs` package)